src/components/AppBar/AppBar.jsx

```diff
@@ -176,19 +197,26 @@ const AppBar = ({
   );
 
   return (
-    <>
-      <Box className={classes.root} data-testid="app-bar">
-        <MaterialAppBar position="fixed" className={classes.appBar}>
-          <Toolbar>
-            {!isLoading ? (<Logo />) : null}
-            <Box className={classes.navLinkBar}>
-              <AddASpace />
-              {userContext.userProfile.username ? <PositionedMenu /> : <LogIn />}
-            </Box>
+    <Box className={isDesktopWidth || !showSearchBar ? classes.root : [classes.root, classes.expandAppBar]} data-testid="app-bar">
+      <MaterialAppBar position="fixed" className={[classes.appBar, classes.muiBreakpointWorkaround]}>
+        <Toolbar className={classes.muiBreakpointWorkaround}>
+          {!isLoading ? (
+            <Logo />
+          ) : null}
+          {(isDesktopWidth && showSearchBar)
```

Comment:
You can just say `isDesktopWidth && showSearchBar && <>...<>` instead of a tertiary with a null here 

![image](https://user-images.githubusercontent.com/1613526/218168210-c7b55dfc-3277-4367-aaf9-849412f77034.png)

Reply:
I went forward with the above changes. I'll make similar changes in the rest of the code base as they come up.